core/src/main/java/nu/yona/server/device/service/DeviceService.java

```diff
@@ -231,6 +232,7 @@ public void deleteDevice(UUID userId, UUID deviceId)
 			UUID deviceAnonymizedId = deviceEntity.getDeviceAnonymized().getId();
 
 			deleteDeviceWithoutBuddyNotification(userEntity, deviceEntity);
+			userAnonymizedService.updateUserAnonymized(userEntity.getAnonymized().getId());
```

Comment:
Does this line fix another bug? If so, should we add another test?

Reply:
See above.